src/main/java/org/variantsync/diffdetective/variation/diff/bad/DiffLineNumberRange.java

```diff
@@ -0,0 +1,6 @@
+package org.variantsync.diffdetective.variation.diff.bad;
```

Comment:
Why is this class in the `bad` package? It doesn't seem to be specific to bad variation diffs.

Reply:
I agree. Moved to `org.variantsync.diffdetective.diff.text` next to the `DiffLineNumber` class.